stable/meteor/templates/secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "{{ include "meteor.name" . }}-secret"
+  namespace: {{ include "meteor.namespace" . }}
+type: Opaque
+data:
+  {{- range $key, $val := .Values.secretConfig -}}
+    {{- (print $key ": " ($val | toString | b64enc | quote )) | nindent 2 -}}
```

Comment:
I'm still thinking if we should we actually pass the already encoded value here 🤔 .
Still weighing the pros and cons.

One concern is when we use this chart on terraform, if we are still passing non-encoded value to this chart, it will be printed when running `tf plan` right? @AkarshSatija

Reply:
@bsushmith